### DISCUSSION TITLE
Allow commenting on multiple occurrence records at once

### DISCUSSION THREAD

**Discussion by ljwalker** [Symbiota/community-board#484] [Wish List]
Requested by Consortium of Midwest Herbaria user:
> Can you make a comment about all instances of a specimen rather than having to leave the same comment for each specimen individually? 
> 
> Example:  There are 8 specimens (at 7 institutions) of [A.A. Heller 5880 (_Alsine glutinosa_)](https://midwestherbaria.org/portal/collections/list.php?collector=Heller&collnum=5880&taxa=Alsine%20glutinosa&usethes=1&taxontype=2) - you want to annotate each of them.  Can one comment be dispersed to all 7 institutions?
>

**Comment by fbungartz** [comment]
Would make sense if collections managers would also be notified per email.
